storops_test/unity/resource/test_snap.py

```diff
@@ -208,3 +208,15 @@ def _inner():
                             description='This is description.',
                             io_limit_policy=None)
         assert_that(_inner, raises(UnityException, "Error Code:0x670166b"))
+
+    @patch_rest
+    def test_snap_restore(self):
+        snap = UnitySnap(cli=t_rest(), _id='38654705785')
+        backup = snap.restore(backup="backup_snap", delete_backup=True)
```

Comment:
No step to check whether the backup_snap is deleted?

Reply:
Cannot do it for now. We may need to enhance the framework, so that we can assert whether the ``snap.delete`` is called or not.